### ISSUE TITLE
[Trigger] CallHub New Call

### ISSUE THREAD

**Issue by aitanagm** [PipedreamHQ/pipedream#5436]
CallHub

I would love to get some help to set up a trigger from CallHub. The idea I have is to create an automation to add a new row in Google Sheets after I call someone using CallHub to get a report of the calls that were made each day. So, I need to find how to create a trigger from CallHub when a new call is made

Any help would be amazing, thanks!

**Comment by sergioeliot2039** [comment]
**App**
Call Hub

**Triggers / Sources**
New Call

additional context.
here are the available webhooks on CallHub, one is "Call Completed", might be useful for the user's scenario
https://support.callhub.io/hc/en-us/articles/8700645870873-What-are-the-available-Triggers-Conditions-and-Actions-on-Workflows-

**Comment by dylburger** [comment]
Thanks for the request! Did you know that anyone can [author and publish their own Pipedream sources](https://pipedream.com/docs/components/quickstart/nodejs/sources/)? Would you be interested in helping out with that? We're happy to answer any questions or support you in any way.

If you don't have the time, would you be willing to reach out to the app provider to see if they're interested in building these? When we work with an app partner, it improves the quality of the integration, and it helps when this kind of request comes from a customer like you.

**Comment by michelle0927** [comment]
@aitanagm I built a source component that creates a webhook and receives events from CallHub. It allows you to select one of four event types. None of them trigger every time a call ends, but when I was testing as an agent, I was required to enter a note when a call ended, and there is an event type (cc.notes) for when an agent adds a note. 

I've added a PR for the new source here (https://github.com/PipedreamHQ/pipedream/pull/5507), but let me know if you think this will work for your use case. There are some `/analytics` endpoints that can be used to retrieve call counts, but they don't return any usefull information about the call.

<img width="568" alt="Screenshot 2023-02-16 at 5 08 27 PM" src="https://user-images.githubusercontent.com/10889229/219498389-4571ed83-c04f-4e7a-ba69-c37acf281813.png">
